regulations/templates/regulations/comment-review-chrome.html

```diff
@@ -3,7 +3,7 @@
 {% block chrome-content %}
 <main role="main">
   <div id="content-body" class="main-content preamble-wrapper comment-review-wrapper">
-    <section id="comment-review" data-page-type="comment-review" data-doc-id="{{doc_number}}">
+    <section id="content-wrapper" class="comment-review" data-page-type="comment-review" data-doc-id="{{doc_number}}">
```

Comment:
+1 to turning this into a `class`. The `content-wrapper` id triggers certain Javascript, though; is it needed, or can we just drop the `id` altogether?